fontforge/fontviewbase.c

```diff
@@ -929,6 +929,52 @@ void FVAddExtrema(FontViewBase *fv, int force_adding ) {
     ff_progress_end_indicator();
 }
 
+void _FVElementAction(FontViewBase *fv, int anysel, void (*func)(SplineChar*, SplineSet*, int), const char* progressmsg) { 
+    int i, cnt=0, layer, first, last, gid;
+    SplineChar *sc;
+    SplineFont *sf = fv->sf;
+    int emsize = sf->ascent+sf->descent;
+
+    for ( i=0; i<fv->map->enccount; ++i )
+	if ( fv->selected[i] && (gid = fv->map->map[i])!=-1 &&
+		SCWorthOutputting(fv->sf->glyphs[gid]) )
+	    ++cnt;
+    ff_progress_start_indicator(10,_(progressmsg),_(progressmsg),0,cnt,1);
+
+    SFUntickAll(fv->sf);
+    for ( i=0; i<fv->map->enccount; ++i ) if ( fv->selected[i] &&
+	    (gid = fv->map->map[i])!=-1 &&
+	    SCWorthOutputting((sc = fv->sf->glyphs[gid])) &&
+	    !sc->ticked) {
+	sc->ticked = true;
+	if ( sc->parent->multilayer ) {
+	    first = ly_fore;
+	    last = sc->layer_cnt-1;
+	} else
+	    first = last = fv->active_layer;
+	for ( layer = first; layer<=last; ++layer ) {
+	    SCPreserveLayer(sc,layer,false);
+	    func(sc, sc->layers[layer].splines, anysel); 
+	}
+	SCCharChangedUpdate(sc,fv->active_layer);
+	if ( !ff_progress_next())
+    break;
+    }
+    ff_progress_end_indicator();
+}
+
+void FVAddInflections(FontViewBase *fv, int anysel) {
+	_FVElementAction(fv,anysel,&SplineCharAddInflections,"Adding points of inflection...");
```

Comment:
Due to how gettext works I'm pretty sure you need to use `_()` here and not in _FVElementAction

Reply:
fixed